sklearn/decomposition/truncated_svd.py

```diff
@@ -93,11 +93,11 @@ class TruncatedSVD(BaseEstimator, TransformerMixin):
     TruncatedSVD(algorithm='randomized', n_components=5, n_iter=7,
             random_state=42, tol=0.0)
     >>> print(svd.explained_variance_ratio_)  # doctest: +ELLIPSIS
-    [ 0.0782... 0.0552... 0.0544... 0.0499... 0.0413...]
+    [ 0.0606... 0.0584... 0.0497... 0.0434... 0.0372...]
```

Comment:
It appears to be the product of a change in `sample_without_replacement` in commit edc9e7. Let me know if there is anything else you want me to change.